ml-agents/mlagents/trainers/learn.py

```diff
@@ -54,7 +54,6 @@ def run_training(run_seed: int, options: RunOptions) -> None:
     Launches training session.
     :param options: parsed command line arguments
```

Comment:
Since you choose to remove `:param run_options:`, could you also change the order of `:param options:` and `:param run_seed:` to match the function argument order?

Reply:
Sure thing, fixed.